src/containers/my-courses/course-search-tool-bar/CourseCreateDescription.tsx

```diff
@@ -0,0 +1,49 @@
+import { styles } from '~/pages/create-or-edit-lesson/CreateOrEditLesson.styles'
+import AppTextField from '~/components/app-text-field/AppTextField'
+import Box from '@mui/material/Box'
+import { useTranslation } from 'react-i18next'
+
+import { TextFieldVariantEnum } from '~/types'
+
+interface CourseCreateDescriptionProps {
+  decriptionName: string
+  titleName: string
+  onChangeHandler: (e: React.ChangeEvent<HTMLInputElement>) => void
+  onChangeHandlerDesc: (e: React.ChangeEvent<HTMLInputElement>) => void
+}
+
+const CourseCreateDescription: React.FC<CourseCreateDescriptionProps> = ({
```

Comment:
You can import just `FC` and use it like this
```js
const CourseCreateDescription: FC<CourseCreateDescriptionProps>
```

Reply:
done